reference/halyard/high-availability.md

```diff
@@ -135,6 +135,14 @@ This service's name when [configuring its sizing](/reference/halyard/component-s
 
 To add a [custom profile](/reference/halyard/custom/#custom-profiles) or [custom service settings](/reference/halyard/custom/#custom-service-settings) for this service, use the name `echo-worker`.
 
+## Deleting Orphan Services
```

Comment:
I would call these "orphaned" services

Reply:
including in the option `--delete-orphaned-services`?